eventhub/endpoint.bal

```diff
@@ -82,14 +86,14 @@ public client class Client {
         return getErrorMessage(response);
     }
 
-    # Update Eventhub properties
+    # Update Event Hub properties.
     #
-    # + eventHubPath - event hub path
-    # + eventHubDescriptionToUpdate - event hub description to update
-    # + return - Return EventHub or Error
+    # + eventHubPath - Event Hub path (Event Hub name)
+    # + eventHubDescriptionToUpdate - EventHubDescriptionToUpdate record with properties to update
+    # + return - EventHub record on success, else returns an error
     @display {label: "Update Event Hub"}
-    remote isolated function updateEventHub(@display {label: "Event hub path"} string eventHubPath, 
-                                            @display {label: "Event hub description to update"} 
+    remote isolated function updateEventHub(@display {label: "Event Hub Path"} string eventHubPath, 
+                                            @display {label: "Event Hub Description to Update"} 
```

Comment:
`Update Description` is enough i guess as the label.

Reply:
Resolved in 4de8b34